src/cephadm/cephadmlib/host_facts.py

```diff
@@ -925,8 +953,6 @@ def _parse_ipv6_route(
         if '/' not in net:  # aggregate /128 mask for single host sub-networks
             net += '/128'
         iface = m[0][1]
-        if iface == 'lo':  # skip loopback devices
-            continue
```

Comment:
I can't remember at this point why we originally had this. I know it was tied to https://tracker.ceph.com/issues/53528 but I don't remember why it was an issue. We should probably investigate that and make sure we aren't recreating an issue before going forward with this